index.html

```diff
@@ -809,185 +809,197 @@ <h3 class="modal-title" id="warningTitle">
-                    </small>
-                  </div>
-                  <div class="col-6">
-                    <label for="abstract">
-                      Abstract for paper.
-                    </label>
-                    <div><a class="btn btn-sm btn-outline-primary" data-bs-toggle="collapse" href="#abstractRow" role="button" aria-expanded="false" aria-controls="abstractRow">Show abstract below ▾</a></div>
-                    <small class="form-text">
-                      An abstract is optional.
-                    </small>
-                  </div>
-                </div>
-                <div id="abstractRow" class="row mt-2 collapse">
-                  <div class="col-8 offset-2">
-                    <label>Abstract</label>
-                    <textarea rows="8" id="talkAbstract" name="talkAbstract" class="form-control" placeholder="Enter abstract here"></textarea>
-                  </div>
-              </form>
-                </div>
-            </div>
+                  <small class="form-text">
+                    Note that these are not checked to fit the time slot.
+                  </small>
+                </div> -->
+              <!-- </div> -->
+              
+              <!-- <div class="row mt-4"> -->
+              <div class="mb-3">
+                <!-- TODO: proposed new title for this category (more streamlined) -->
+                <label for="paperId">
+                  HotCRP Paper ID
+                </label>
+                <!-- TODO: shouldn't this be input type num or something? -->
+                <input type="text" id="paperId" name="paperId" class="form-control my-1 w-25" autocomplete="off" />
+                <small class="form-text">
+                  This is intended to contain the ID from HotCRP, and must be unique.
+                </small>
+              </div>
+            </form>
+
+            <!-- TODO: should we be using two modal body classes in one modal? eek -->
             <div id="deleteTalkWarning" class="modal-body">
               <div class="alert alert-danger">If you delete this talk, then you cannot recover the data. It will be permanently removed from the app.</div>
             </div>
-            <div class="modal-footer">
-              <button id="talkDeleteButton" type="button" class="btn btn-danger btn-sm float-start" onclick="deleteTalk()">
-                Delete talk completely
-              </button>
-              <button type="button" class="btn btn-light btn-sm" data-bs-dismiss="modal">
-                Cancel
-              </button>
-              <button type="button" class="btn btn-success btn-sm" onclick="saveTalk()">
-                Save Talk
-              </button>
-            </div>
+          </div>  <!-- END modal-body -->
+          <div class="modal-footer">
+            <button id="talkDeleteButton" type="button" class="btn btn-danger btn-sm float-start" onclick="deleteTalk()">
+              Delete talk completely
```

Comment:
We need to change the deleteTalk() javascript method to scroll down to show the warning when it is clicked the first time. Since we are using jquery, the method is:
```
$('#deleteTalkWarning').show();                                                                                                                                           
$('#deleteTalkWarning')[0].scrollIntoView({behavior: 'smooth', block: 'end'}); 
```